plugin/storage/es/factory.go

```diff
@@ -209,6 +208,29 @@ func createSpanWriter(
 	return writer, nil
 }
 
+func createDependencyReader(
+	logger *zap.Logger,
+	client es.Client,
+	cfg config.ClientBuilder,
+) (dependencystore.Reader, error) {
+
+	dependenciesMappings := GetDependenciesMappings(cfg.GetNumShards(), cfg.GetNumReplicas(), client.GetVersion())
+	reader := esDepStore.NewDependencyStore(esDepStore.DSParams{
+		Client:              client,
+		Logger:              logger,
+		IndexPrefix:         cfg.GetIndexPrefix(),
+		UseReadWriteAliases: cfg.GetUseReadWriteAliases(),
+	})
+	if cfg.IsCreateIndexTemplates() {
```

Comment:
The reader does not have to create index templates

Reply:
I shall remove that with the next push then.